src/app/(infoscreen)/@restaurants/page.tsx

```diff
@@ -0,0 +1,39 @@
+import { Slide } from '@/components/Carousel';
+import { RestaurantMenu } from '@/components/Restaurants/Restaurant';
+import { getRestaurants } from '@/server/restaurants';
+
+const Restaurants = async () => {
+	const restaurants = await getRestaurants([
+		'Chemicum',
+		'Exactum',
+		'Kaivopiha',
+	]);
+
+	console.dir(Object.values(restaurants), { depth: 3 });
+
+	return (
+		<Slide full className="bg-green-900 pb-0">
+			{Object.values(restaurants).length === 0 ? (
+				<p>There are no open restaurants at the moment.</p>
+			) : (
+				<div className="flex min-h-0 overflow-hidden">
+					<div className="flex">
+						{restaurants.Chemicum && (
```

Comment:
if `restaurants` is sorted, this can be just a restaurants.map or similar (with Object.entries?)

Reply:
also scales better, only need to add a new restaurant to service instead of modifying multiple places to include it in listing